File: tests/expectations/both/monomorph-2.c
#include <stdint.h>
#include <stdlib.h>
#include <stdbool.h>

typedef struct A A;

typedef struct B B;

typedef struct List_B {
  B *members;
  uintptr_t count;
} List_B;

typedef struct List_A {
  A *members;
  uintptr_t count;
} List_A;

void bar(List_B b);

void foo(List_A a);

File: tests/expectations/monomorph_2.zig
const std = @import("std");

const A = opaque {};

const B = opaque {};

pub const List_A = extern struct {
    _members: ?*A,
    count: usize,
};

pub const List_B = extern struct {
    _members: ?*B,
    count: usize,
};

extern fn foo(a: List_A) anyopaque;

extern fn bar(b: List_B) anyopaque;
